package.json

```diff
@@ -53,6 +53,11 @@
       "import": "./const.mjs",
       "require": "./const.js"
     },
+    "./types": {
+      "types": "./types.d.ts",
+      "import": "./types.mjs",
+      "require": "./types.js"
```

Comment:
Can we rather scope by packages instead by file types? It's not cohesive.

We can have `import {APL, ...} from '@saleor/app-sdk/APL'` and avoid mixing thing around the project.

The same I did in #25